tests/unit/build/CMakeLists.txt

```diff
@@ -55,11 +55,8 @@ function(pika_create_cmake_test name using_install_dir pika_dir build_type
       --output-on-failure
     VERBATIM
```

Comment:
Sorry, one more: is this a valid option to `add_test`? I think it's an option to `add_custom_target`.